app/src/main/java/com/prangesoftwaresolutions/audioanchor/MediaPlayerService.java

```diff
@@ -681,8 +681,10 @@ void play() {
         // Request a partial wake lock for the duration of the playback
         mLockManager.acquireWakeLock();
 
-        if (mMediaPlayer != null && !mMediaPlayer.isPlaying() && (autoplay || getCurrentPosition() != getDuration())) {
```

Comment:
The `or` condition ( `|| getCurrentPosition() != getDuration()` ) needs to stay there. Otherwise it's not possible to start playback at all when autoplay is disabled.

Reply:
You are right, I have restored the original condition.